recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
     - python
     - wheel
     - setuptools
+    - pytorch {{ pytorch }}
```

Comment:
I wonder if we need it for building the package. I didn't find any evidence of that. See also conda-forge https://github.com/conda-forge/sentence-transformers-feedstock/blob/main/recipe/meta.yaml#L18-L21

Reply:
*Technically* not needed for building, but linter is complaining. Should I ignore it?